<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveShortPathForm</class>
 <widget class="QWidget" name="SaveShortPathForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save short path</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Short path</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Start node:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="startNodeEdit">
            <property name="toolTip">
             <string>Node id for start writing bracket style paths.</string>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="maxLength">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Path weight limit:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="pathLimitEdit">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Path weight limit. It's nesting weight limiter. &lt;/p&gt;&lt;p&gt;Useful if need crop graph or give some graph part that &lt;br/&gt;accessible in not larger than &amp;quot;path weight limit&amp;quot; edges weight sum.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string notr="true">0</string>
            </property>
            <property name="maxLength">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="printIndentsCheck">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;
&lt;p&gt;Print indents for more readable data. Used in bracket style.&lt;br/&gt;
Each new nesting increase indent lewel.&lt;/p&gt;
&lt;p&gt;
Example:
&lt;table&gt;
&lt;tr&gt;
&lt;td&gt;
Turned off:&lt;br&gt;
1(2[3](3[5](4[9] 5[6]) 7[4]))
&lt;/td&gt;
&lt;td&gt;
Turned on:&lt;br&gt;
1(&lt;br&gt;
&amp;middot; &amp;middot; 2[3](&lt;br&gt;
&amp;middot; &amp;middot; &amp;middot; &amp;middot; 3[5](&lt;br&gt;
&amp;middot; &amp;middot; &amp;middot; &amp;middot; &amp;middot; &amp;middot; 4[9]&lt;br&gt; 
&amp;middot; &amp;middot; &amp;middot; &amp;middot; &amp;middot; &amp;middot; 5[6]&lt;br&gt;
&amp;middot; &amp;middot; &amp;middot; &amp;middot; ) &lt;br&gt;
&amp;middot; &amp;middot; &amp;middot; &amp;middot; 7[4]&lt;br&gt;
&amp;middot; &amp;middot; )&lt;br&gt;
)
&lt;/td&gt;
&lt;/tr&gt;
&lt;/table&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Print indents</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Example</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="exampleLabel">
          <property name="text">
           <string notr="true">Example</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string notr="true">info</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
